docs/src/integer.md

```diff
@@ -362,6 +362,7 @@ moebius_mu(::fmpz)
 ```@docs
 jacobi_symbol(::Int, ::Int)
 jacobi_symbol(::fmpz, ::fmpz)
+kronecker_symbol(::fmpz, ::fmpz)
```

Comment:
There is no docstring for this. We could attach the docstring to
```
function kronecker_symbol end
```
Then this should work.

Reply:
Oh, I'll just correct the place where it is instantiated.